planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/data_manager.cpp

```diff
@@ -84,7 +80,7 @@ LaneChangerParameters DataManager::getLaneChangerParameters() {return parameters
 bool DataManager::getLaneChangeApproval()
 {
   constexpr double timeout = 0.5;
-  if (clock_->now() - lane_change_approval_.stamp > rclcpp::Duration::from_seconds(timeout)) {
+  if (clock_->now() - lane_change_approval_.stamp > rclcpp::Duration(timeout)) {
```

Comment:
I guess it's better to use from_seconds.
https://github.com/ros2/rclcpp/blob/09950ba23fa2c3e1a04d1b667c91c6fcc1e00988/rclcpp/include/rclcpp/duration.hpp#L132-L134

Reply:
Fixed.